applications/ExternalSolversApplication/external_includes/feast_solver.h

```diff
@@ -101,9 +101,9 @@ class FEASTSolver: public LinearSolver<TSparseSpaceType, TDenseSpaceType,
             }
         })");
 
-        mpParam->RecursivelyValidateAndAssignDefaults(default_params);
+        mParam.RecursivelyValidateAndAssignDefaults(default_params);
 
-        if (mpParam->GetValue("linear_solver_settings")["solver_type"].GetString() != "complex_skyline_lu_solver")
+        if (mParam.GetValue("linear_solver_settings")["solver_type"].GetString() != "skyline_lu")
```

Comment:
i review myself since a lot of time passed: should this be complex_skyline_lu_solver?